src/core/Akka/Actor/Futures.cs

```diff
@@ -117,66 +117,44 @@ public static Task<T> Ask<T>(this ICanTell self, object message, TimeSpan? timeo
         /// This exception is thrown if the system can't resolve the target provider.
         /// </exception>
         /// <returns>TBD</returns>
-        public static async Task<T> Ask<T>(this ICanTell self, Func<IActorRef,object> messageFactory, TimeSpan? timeout, CancellationToken cancellationToken)
+        public static async Task<T> Ask<T>(this ICanTell self, Func<IActorRef, object> messageFactory, TimeSpan? timeout, CancellationToken cancellationToken)
         {
             await SynchronizationContextManager.RemoveContext;
```

Comment:
Not sure if this is needed - can tell the CTRs and everything else not to run in synchronization context if necessary.